src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -361,6 +361,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 
 	private addDecoration(range: NotebookRange): void {
 		if (range && this.output && this.output.nativeElement) {
+			let markDoc = new Mark(this.output.nativeElement);
```

Comment:
It might be helpful to explain in a comment why the two different Mark objects are needed